internal/ethapi/api.go

```diff
@@ -1522,6 +1548,54 @@ func (s *PublicTransactionPoolAPI) SendRawTransaction(ctx context.Context, encod
 	return SubmitTransaction(ctx, s.b, tx)
 }
 
+// SendBlockBatches will:
+// * Verify the batches are signed by the BlockBatchesSender
+// * Update the Geth timestamp to the provided timestamp
+// * handle the RollupTransaction Batches contained in the provided Block atomically
+func (s *PublicTransactionPoolAPI) SendBlockBatches(ctx context.Context, messageAndSig []hexutil.Bytes) []error {
+	if len(messageAndSig) != 2 {
+		return []error{fmt.Errorf("Incorrect number of arguments. Expected 2, got %d", len(messageAndSig))}
+	}
+	if !crypto.VerifyMessageSignature(s.b.ChainConfig().BlockBatchesSender.Bytes(), messageAndSig[0], messageAndSig[1]) {
+		return []error{fmt.Errorf("Signature does not match Block Batch Sender address %x", s.b.ChainConfig().BlockBatchesSender)}
+	}
+	var blockBatches BlockBatches
+	if err := json.Unmarshal(messageAndSig[0], &blockBatches); err != nil {
+		return []error{fmt.Errorf("Incorrect format for BlockBatches type. Received: %s", messageAndSig[0])}
+	}
+
+	txCount := 0
+	signer := types.MakeSigner(s.b.ChainConfig(), s.b.CurrentBlock().Number())
+	key, _ := crypto.GenerateKey()
```

Comment:
As we discussed on the call, it probably makes sense to make this a single signing key across all synced rollup transactions